### PR TITLE
feat: hero overlay variant

### LINKED ISSUE #31: hero layout for services page
Create hero for services page:

Screenshot 2023-07-27 at 6 12 00 PM

### PR DESCRIPTION
**Pull request by mfrisbey**
Hero variant for the services page.

Fix #31

Test URLs:
- Before: https://main--netjets-qssecurity--hlxsites.hlx.page/executive-protection-services
- After: https://issue-31--netjets-qssecurity--hlxsites.hlx.page/executive-protection-services

### CODE REVIEW

**Comment by mfrisbey** [comment]
Looking into the performance and CLS scores.

**Comment by mfrisbey** [comment]
Looks like it was just a bad run.

File: blocks/hero/hero.css [hunk 1]
```diff
@@ -59,6 +59,35 @@ main .hero.short h2 {
...
+  padding: 45px 40px 0;
+  gap: 5px;
+}
+
+main .hero.overlay h2 {
+  font-size: var(--heading-font-size-xxs);
+  line-height: 22px;
+}
+
+main .hero-wrapper:has(.hero.overlay) picture {
```
Comment (ravkiran) [inline blocks/hero/hero.css:85]: `:has` isn't supported in firefox https://caniuse.com/?search=has
so the layout is broken in firefox
image

Comment (mfrisbey) [reply blocks/hero/hero.css:85]: @ravkiran sure, I'll fix this in conjunction with the other issue.

**Review by ravkiran — APPROVED** [review]
firefox issue will be fixed in another PR, so approving this.

**Pull request merged**